parsec/core/cli/share_workspace.py

```diff
@@ -13,24 +13,36 @@
 WORKSPACE_ROLE_CHOICES = {"NONE": None, **{role.value: role for role in WorkspaceRole}}
 
 
-async def _share_workspace(config, device, name, user_id, user_role):
+async def _share_workspace(config, device, name, user_id, recipiant, user_role):
+    if recipiant and user_id or not recipiant and not user_id:
+        raise click.ClickException("Either recipiant or user_id should be used, but not both")
```

Comment:
Should indicate the option name used by the user for better understanding: "[...] --recipiant or --user-id [...]"